src/python/app.py

```diff
@@ -0,0 +1,60 @@
+import json
```

Comment:
Ronan.  I would add a small piece of commentary to each of these files explaining what their roles and responsibilities are.  Otherwise, you are leaving the user to guess what is going on.